.travis.yml

```diff
@@ -1,13 +0,0 @@
-dist: xenial # required for Python >= 3.7
```

Comment:
Let's keep the travis setup for some time till we see no issues with prow presubmit, probably for a couple of weeks.